InventorySales/Model/AreProductsSalableForRequestedQty/AreProductSalableResult.php

```diff
@@ -0,0 +1,85 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventorySales\Model\AreProductsSalableForRequestedQty;
+
+use Magento\Framework\Api\AttributeValueFactory;
+use Magento\Framework\Api\ExtensionAttributesFactory;
+use Magento\Framework\Data\Collection\AbstractDb;
+use Magento\Framework\Model\AbstractExtensibleModel;
+use Magento\Framework\Model\Context;
+use Magento\Framework\Model\ResourceModel\AbstractResource;
+use Magento\Framework\Registry;
+use Magento\InventorySalesApi\Api\Data\AreProductsSalableResultExtensionInterface;
+use Magento\InventorySalesApi\Api\Data\AreProductsSalableResultInterface;
+
+/**
+ * @inheritDoc
+ */
+class AreProductSalableResult extends AbstractExtensibleModel implements AreProductsSalableResultInterface
```

Comment:
Better to use`ProductsSalableResult` name here

Reply:
@sidolov, renamed